.rubocop.yml

```diff
@@ -96,3 +114,7 @@ UnneededInterpolation:
 
 UselessAccessModifier:
   Enabled: false
+
+Lint/HandleExceptions:
```

Comment:
Suppressing exceptions is quite a bad thing even in specs. Can you fix the code rather than disable the rule? E.g. You can add some logging to the rescue block.